frontend/src/lib/components/ShowValueFilter/showValueFilterLogic.ts

```diff
@@ -0,0 +1,65 @@
+import { kea } from 'kea'
+import { objectsEqual } from 'lib/utils'
+import { InsightLogicProps, InsightType, TrendsFilterType } from '~/types'
+import { keyForInsightLogicProps } from 'scenes/insights/sharedUtils'
+import { insightLogic } from 'scenes/insights/insightLogic'
+import { isTrendsFilter } from 'scenes/insights/sharedUtils'
+import { TrendsFilter } from '~/queries/schema'
+import { insightDataLogic } from 'scenes/insights/insightDataLogic'
+import { filterForQuery } from '~/queries/utils'
+
+import type { showValueFilterLogicType } from './showValueFilterLogicType'
+
+export const showValueFilterLogic = kea<showValueFilterLogicType>({
```

Comment:
As I understood this is old kea (v2) format - new format uses arrays https://keajs.org/docs/meta/key/

Reply:
Ah, good spot. Copied another logic and didn't notice